Bump github/drone/go-scm to v1.31.0

cod3rboy opened the pull request:
# Change
- Bump `github.com/drone/go-scm` dependency to `v1.31.0` to fix organization secret access for Gitea. 
PR: https://github.com/drone/go-scm/pull/263

- Updated mocks with `go generate` to fix unimplemented method error during test run.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/harness/drone?pullRequest=3354) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/harness/drone?pullRequest=3354) before we can accept your contribution.<br/>**1** out of **3** committers have signed the CLA.<br/><br/>:white_check_mark: johannesHarness<br/>:x: eltociear<br/>:x: vmorganp<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/harness/drone?pullRequest=3354) it.</sub>

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/harness/drone?pullRequest=3354) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/harness/drone?pullRequest=3354) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/harness/drone?pullRequest=3354) it.</sub>

johannesHarness committed:
remove all drone files but .github and .github_changelog_generator

johannesHarness committed:
ADD GITNESS

johannesHarness committed:
remove drone readme.md

johannesHarness committed:
disable export, fix default pipeline

johannesHarness committed:
updated readme

johannesHarness committed:
remove leftover file

johannesHarness committed:
Add Git Code Hosting to Drone #3364 

LET'S.GO.[DRONE](https://www.drone.io/).[GITNESSSSSS](https://gitness.com/)!!! <3

johannesHarness committed:
Update README.md with Drone Information.

johannesHarness committed:
Update LICENSE.md with proper year + company details

johannesHarness committed:
minor fixing based on first feedback :) (#3365)

johannesHarness committed:
[MISC] Adding Docker Instructions to README (#3366)

johannesHarness committed:
[BUILD] Remove internal reference (#3377)

eltociear committed:
[MISC] Fix typo in README.md (#3379)

vmorganp committed:
Fix typo in PluginsPanel.tsx (#3383)

johannesHarness committed:
soft-delete branch 'master' due to move of Gitness to 'main' branch, Drone to 'drone' branch. Add readme for existing clones/forks to provide context.

bot2-harness commented:
This pull request has been automatically closed due to 30 days of inactivity. If you believe this was done in error, please feel free to reopen the PR or reach out to the BT team for further assistance. Thank you for your contributions!
